api-reference/v1.0/toc.yml

```
@@ -5081,6 +5081,13 @@ items:
               href: api/certificatebasedauthconfiguration-get.md
             - name: Delete
               href: api/certificatebasedauthconfiguration-delete.md
+          - name: Password
+            href: resources/passwordauthenticationmethod.md
+            items:
+            - name: Reset
```

Surface the List and Get files in the TOC
```suggestion
            - name: List
              href: api/authentication-list-passwordmethods.md
            - name: Get
              href: api/passwordauthenticationmethod-get.md
            - name: Reset
```